modules/customize-certificates-add-service-serving-configmap.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-configmap-{context}"]
+= Add a service certificate to a ConfigMap
+
+To secure communication to services that consume a ConfigMap, 
+have the cluster generate a signed serving certificate and key pair into the
+ConfigMap. To do this, set the 
+`service.beta.openshift.io/inject-cabundle=true` 
+annotation on your ConfigMap. A certificate will be generated, which allows
+consumers to trust the `service-ca.crt` in their TLS client configuration,
+allowing connections to services that utilize these service sharing 
+certificates.
+
+.Procedure
+
+. Annotate the ConfigMap with `service.beta.openshift.io/inject-cabundle=true`.
++
+----
+$ oc annotate configmap <configmap-name> service.beta.openshift.io/inject-cabundle=true
+----
++
+[NOTE]
+====
+Explicitly referencing the `service-ca.crt` key in a volumeMount will 
```

Comment:
If the ConfigMap's certificate is referenced by a Pod's volumeMount, then the Pod will not start until ConfigMap has the necessary certificate. I think it's important to note here so that customers know if they're performing these steps it may cause a delay in Pod creation/start. This should happen automatically and without issue to the end user.